### ISSUE TITLE
Fuck sucks, it can not even fix itself (:

### ISSUE THREAD

**Issue by kirushyk** [nvbn/thefuck#446]
Just installed thefuck via homebrew but...

```
$ fuck
Seems like fuck alias isn't configured!
Please put eval $(thefuck --alias) in your ~/.bashrc.
More details - https://github.com/nvbn/thefuck#manual-installation
$ fuck
Seems like fuck alias isn't configured!
Please put eval $(thefuck --alias) in your ~/.bashrc.
More details - https://github.com/nvbn/thefuck#manual-installation
```

I expected second time it will work!

**Comment by scorphus** [comment]
Good catch, @kirushyk! I can code that rule!

**Comment by kirushyk** [comment]
Thank you, because I don't want to manually tweak my .bashrc...

**Comment by scorphus** [comment]
Count on us! :wink:

**Comment by minhoryang** [comment]
+1 GO F!

**Comment by larsgrefer** [comment]
:+1:

**Comment by nvbn** [comment]
Implemented in #611 =)
